test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerValidationParametersTests.cs

```diff
@@ -13,6 +13,8 @@
 using Microsoft.IdentityModel.Tokens;
 using Xunit;
 
+#nullable enable
+
```

Comment:
be consistent with spacing around #nullable.
Other files do not have a blank line.